inconspiquous/dialects/qu.py

```diff
@@ -1,83 +1,162 @@
 from typing import ClassVar
-from xdsl.ir import Dialect, ParametrizedAttribute, TypeAttribute
+
+from xdsl.dialects.builtin import IntAttr, IntAttrConstraint
+from xdsl.ir import (Dialect, ParametrizedAttribute, TypeAttribute, SSAValue,
+                   OpResult, Attribute)
 from xdsl.irdl import (
-    AnyInt,
-    IRDLOperation,
-    IntVarConstraint,
-    RangeOf,
-    irdl_attr_definition,
-    irdl_op_definition,
-    prop_def,
-    var_result_def,
-    eq,
-)
+    AnyInt, IRDLOperation, IntVarConstraint, ParamAttrConstraint, RangeOf, eq,
+    irdl_attr_definition, irdl_op_definition, operand_def, prop_def,
+    result_def, var_operand_def, var_result_def, ParameterDef, Operand,
+    VarOperand, VarOpResult)
+from xdsl.utils.exceptions import VerifyException
+from xdsl.parser import AttrParser
+from xdsl.printer import Printer
 
 from inconspiquous.alloc import AllocAttr
 from inconspiquous.constraints import SizedAttributeConstraint
 
 
 @irdl_attr_definition
 class BitType(ParametrizedAttribute, TypeAttribute):
-    """
-    Type for a single qubit
-    """
-
+    """Type for a single qubit."""
```

Comment:
Would it be possible to revert the small diffs to parts of code which are not touched by this PR? It will make the diff much easier to read.